### PR TITLE
Changed default installed tf gpu version to 2.0

### PR DESCRIPTION
**Pull request by twinkarma**
Added tf versions to compatibility matrix

### CODE REVIEW

File: sharc/software/apps/tensorflow.rst [hunk 1]
```diff
@@ -120,11 +120,17 @@ Tensorflow releases depend on specific versions of both CUDA and CUDNN. If the w
...
 
-The following table shows the which module to load for the various versions of Tensorflow, based on the `tested build configurations <https://www.tensorflow.org/install/source#linux>`_. Newer versions may require more recent CUDA and CUDNN releases. 
+The following table shows the which module to load for the various versions of Tensorflow, based on the `tested build configurations <https://www.tensorflow.org/install/source#linux>`_. 
+
+
 
 +------------+------+--------+--------------------------------------------+
 | Tensorflow | CUDA | CUDNN  | Module                                     | 
 +============+======+========+============================================+
+| >= 2.0.0  | 10.0  | >= 7.4 | `libs/cudnn/7.5.0.56/binary-cuda-10.0.130` |
```
Comment (willfurnass) [inline sharc/software/apps/tensorflow.rst:130]: Does tensorflow 2.1.0 require CUDA 10.1?

Comment (twinkarma) [reply sharc/software/apps/tensorflow.rst:130]: Yes it does

Comment (twinkarma) [reply sharc/software/apps/tensorflow.rst]: I've removed the `>=` for the latest three versions but I don't really want to add 2.1.0 yet as we don't have a CUDA 10.1 module

Comment (ptheywood) [reply sharc/software/apps/tensorflow.rst]: Would it not be useful to say that is the case but this is  not yet available?

Comment (willfurnass) [reply sharc/software/apps/tensorflow.rst]: Added TF 2.1 requirements in PR #1002

**Pull request merged**